schemas/examples/biologic_mpr.yml

```diff
@@ -1,20 +1,20 @@
 ---
-id: >-
-    biologic-mpr
-name: >-
-    BioLogic MPR file
-description: >-
-    A binary data file created by BioLogic cyclers, containing time- and frequency-domain
-    data pertaining to many electrochemical techniques, such as voltammetry,
-    potentiometry, coulometry and impedance spectroscopy.
-associated_vendors:
-    - BioLogic
-subject:
-    - electrochemistry
-    - voltammetry
-    - impedance spectropscopy
-    - potentiometry
-associated_instruments:
-    - BioLogic MPG-200
-associated_software:
-    - BioLogic EC-Lab
+    - id: >-
+          biologic-mpr
+      name: >-
+          BioLogic MPR file
+      description: >-
+          A binary data file created by BioLogic cyclers, containing time- and frequency-domain
+          data pertaining to many electrochemical techniques, such as voltammetry,
+          potentiometry, coulometry and impedance spectroscopy.
+      associated_vendors:
+          - BioLogic
+      subject:
+          - electrochemistry
+          - voltammetry
+          - impedance spectropscopy
+          - potentiometry
+      associated_instruments:
+          - BioLogic MPG-200
+      associated_software:
+          - BioLogic EC-Lab
```

Comment:
Is an indented `yml` like this valid?

Reply:
This was the cause of the test failure, serves me right for trying to use the GitHub web editor :upside_down_face: I've reverted this with the container change anyway.